.gitignore

```diff
@@ -2,4 +2,3 @@
 .bin
 out/
 /site/node_modules/.cache
-/site/node_modules
```

Comment:
the entire `node_modules` should be ignored, there's no reason to retain the entire copies of dependencies checked in the repo here